profiles/schema.py

```diff
@@ -50,10 +56,60 @@ def resolve_type(self, info, **kwargs):
     )
 
 
+class ProfilesConnection(graphene.Connection):
+    class Meta:
+        abstract = True
+
+    count = graphene.Int()
+    total_count = graphene.Int()
+
+    def resolve_count(self, info):
+        return self.length
+
+    def resolve_total_count(self, info):
+        return self.iterable.model.objects.filter(
+            service__service_type=SERVICE_TYPES[1][0]
+        ).count()
+
+
+class ProfileFilter(FilterSet):
+    class Meta:
+        model = Profile
+        fields = ("first_name", "last_name", "nickname", "email", "phone", "language")
+
+    first_name = CharFilter()
```

Comment:
I actually meant `exact` makes sense for the `language` field, where you have only codes and it is probably best to get the code that matches excatly. other fields might be quite useful to be `icontains`.